operations/observability/mixins/cross-teams/rules/components/nodes/alerts.libsonnet

```
@@ -29,6 +29,38 @@
                 ) - 10 < 5
               |||,
           },
+          {
+            alert: 'GitpodNodeConntrackTableIsFull',
+            labels: {
+              severity: 'critical',
+            },
+            'for': '5m',
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodNodeConntrackTableIsFull.md',
+              summary: 'Node conntrack table is almost full',
+              description: 'Node {{ $labels.node }} conntrack table is almost full. If it gets full, packets will be getting dropped.',
+            },
+            expr:
+              |||
+                (node_nf_conntrack_entries / on (pod) node_nf_conntrack_entries_limit / on (pod) group_right kube_pod_info) > 0.95
+              |||,
+          },
+          {
+            alert: 'GitpodNodeConntrackTableGettingFull',
+            labels: {
+              severity: 'warning',
+            },
+            'for': '10m',
+            annotations: {
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodNodeConntrackTableIsFull.md',
+              summary: 'Node conntrack table is getting full',
+              description: 'Node {{ $labels.node }} conntrack table is getting full. If it gets full, packets will be getting dropped.',
+            },
+            expr:
+              |||
+                (node_nf_conntrack_entries / on (pod) node_nf_conntrack_entries_limit / on (pod) group_right kube_pod_info) > 0.80
```

```suggestion
               (node_nf_conntrack_entries / node_nf_conntrack_entries_limit) > 0.80
```

Any reason to not use a simplified query?

A join can make the query quite heavy and the added label, pod, doesn't provide any info, right? 🙂 
For a node that is running out of conntrack entries, there is no need to know which exporter pod is running, I believe you only care about what node is running out of entries

---

Yes, you are 100% right. Copy-paste error while testing this in grafana. Fixed!